ui/components/MeshModelRegistry/MeshModelComponent.js

```diff
@@ -104,10 +104,10 @@ const MeshModelComponent = ({
         model.components = components;
         model.relationships = relationships;
       });
-
       await Promise.all(componentPromises);
```

Comment:
@aabidsofi19 Promise == bad, yes? // @ptdatta

Reply:
RTK can be a better option